src/mpcCoreKit.ts

```diff
@@ -1506,7 +1508,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
       clientShareAdjustedHex,
       pubKeyHex,
       data,
-      serverCoefficientsHex
+      serverCoefficientsHex,
+      this.socketTransports
```

Comment:
Wasn't there another parameter here?

Reply:
that's on another version of tss client which is not in production yet.